glide.lock

```diff
@@ -52,46 +24,29 @@ imports:
 - name: github.com/decred/dcrutil
   version: 0484582bf5503574d824f110e836a8c48aa60c8c
   subpackages:
-  - base58
   - hdkeychain
 - name: github.com/decred/dcrwallet
   version: 1160e7671c476b23a1fa89f0dac19b295dd7c2b2
   subpackages:
-  - internal/zero
-  - pgpwordlist
-  - snacl
   - waddrmgr
-  - walletdb
-- name: github.com/decred/ed25519
-  version: b0909d3f798b97a03c9e77023f97a5301a2a7900
-  subpackages:
-  - edwards25519
 - name: github.com/go-sql-driver/mysql
   version: 8f6c67f0228a4a6a962008d6c3d1e694e4532e71
-- name: github.com/go-utils/uslice
-  version: 7f8c3df848991b2e38824a7bda55bda1339b651e
 - name: github.com/gorilla/context
   version: 08b5f424b9271eedf6f9f0ce86cb9396ed337a42
-- name: github.com/gorilla/securecookie
```

Comment:
Not sure why glide is removing this.  It's a dependency of github.com/gorilla/sessions.